bigbluebutton-html5/imports/ui/components/external-video-player/external-video-player-graphql/modal/component.tsx

```diff
@@ -72,6 +74,8 @@ const ExternalVideoPlayerModal: React.FC<ExternalVideoPlayerModalProps> = ({
         externalVideoUrl = `https://${m[1]}/Podcast/Social/${m[3]}.mp4`;
       }
     }
+    // stop pinned notes so they don't interfere with the video
+    pinSharedNotes({ variables: { pinned: false } });
```

Comment:
Maybe we shouldn't depend on the client for that. It feels like something that should be handled on the backend. What do you think?

Reply:
https://github.com/bigbluebutton/bigbluebutton/pull/23405